Netlink.cpp

```diff
@@ -83,10 +83,17 @@ int Netlink::Send(uint16_t type, const void* data, size_t len, reply_fn_t&& repl
     nlmsghdr *nl = reinterpret_cast<nlmsghdr*>(_data.data());
     nl->nlmsg_type = type;
     nl->nlmsg_flags = NLM_F_REQUEST|NLM_F_ACK;
-    nl->nlmsg_seq = _sequence++;
-    if (_sequence == 0) {
-        _sequence = 1;
-    }
+
+    // Make sure the seq is unique and not in use
+    uint32_t seq = 0;
+    do {
+        seq = _sequence++;
+        if (_sequence == 0) {
+            _sequence = 1;
+        }
```

Comment:
Do we still need this?

Reply:
seq==0 is a special case, so yes, it is possible for _sequence to wrap and if it does, we need skip _sequence==0